### PR TITLE
[layerzero oft] add configs for linea

### PR DESCRIPTION
**Pull request by zhangchitc**
- removed merlin chain configs
- add linea network and connections with ethereum and solana

## Mainnet deployment

- Linea OFT address: [`0xF3df0A31ec5EA438150987805e841F960b9471b6` ](https://lineascan.build/address/0xF3df0A31ec5EA438150987805e841F960b9471b6)

```
30101 ethereum
30168 solana
30183 linea
```

- ethereum->linea:
`npx hardhat --network ethereum send --dst-eid 30183 --amount 1000000000000000000 --to 0xc031C368b51c28266396273b0C6ce2489b00969d`
https://layerzeroscan.com/tx/0xbc47d30e79dd51b2e843ad50e7ed3eb7b5bedde4541559eb22d013d35feda370

- linea->solana:
`npx hardhat --network linea send --dst-eid 30168 --amount 1000000000000000000 --to BhTpGbF7gBUiGjESQHVoBNhx2iy7Y21L2hAhRPdNT5ZF`
https://layerzeroscan.com/tx/0x5f8521d38298ac4550ff91062c76c5bf7e1ceedfff2fc9d821cae8a1adad2831

- solana->linea:
`npx hardhat lz:oft:solana:send --amount 1000000000 --from-eid 30168 --to 0xc031C368b51c28266396273b0C6ce2489b00969d --to-eid 30183 --mint Dz8VutERqbHR2aFL5A3s1Ky4dG1unJT1jUFXXPaY9ytX --program-id woo98ny1QLULqdTzpNM8PiJpwfzL5MJ9pAmLw1rfvk7 --escrow 22ag34UbSzp2d4ekBLkekLHX1woPLDYZHqLGPY7e3ybu`
https://layerzeroscan.com/tx/4ZKj7W7ipo7dxHQVNzZbNgZD9S8NFgjiWpV5gvbPgpzPMXvd5QwDneik3uyiKnxzjC97LPaJpK85gpfnkbn45MUd

- linea->ethereum:
`npx hardhat --network linea send --dst-eid 30101 --amount 1000000000000000000 --to 0xc031C368b51c28266396273b0C6ce2489b00969d`
https://layerzeroscan.com/tx/0x6fc6268f91744ef457ef99f1dacb6926414db85f5dead1b6ed331374c2701eee

## Devnet testing

```
WOO On sepolia: 	https://sepolia.etherscan.io/address/0xc9fa9efa738AFd3504cB59547114F0847C44A8fa


9GCwSyTCCEWhh5wD59Q84PEnevEmvdeaQbYatSnYBR2s
C57J7hTDbQPLQWL7hwhHcDwnK1URVbVzwjDcFtUjc4eq

pnpm hardhat lz:oft:solana:create --eid 40168 --program-id C57J7hTDbQPLQWL7hwhHcDwnK1URVbVzwjDcFtUjc4eq --only-oft-store true --name WOO --symbol WOO

{
    "programId": "C57J7hTDbQPLQWL7hwhHcDwnK1URVbVzwjDcFtUjc4eq",
    "mint": "5my4h8qULDHJwxAVcKwk9n64gFz5dm3ridpzFTns6t8H",
    "mintAuthority": "F8LUitByLBzhRfJ6y8SxoEH9P2Zygjicu2sBaY4m3KDX",
    "escrow": "8MhP39SfLNWbLJVBCNNUU49z2CvRHfhqcj5uR57ZSWCZ",
    "oftStore": "FqJAVaFEsnwfAMfuPYB9NsWyBEP8Xs5buNhG5KQAaWjv"
}

WOO Adapter on sepolia: https://sepolia.etherscan.io/address/0x053b86b64774ca0B44b4289c4D49FCE4D3EE65cC#code

WOO OFT on Linea Sep: https://sepolia.lineascan.build/address/0xe121aD2DA0fEe579D30397ed969110A1358Ea761


npx hardhat lz:oapp:init:solana --oapp-config layerzero.config.ts --solana-secret-key <solana_secret_key> --solana-program-id C57J7hTDbQPLQWL7hwhHcDwnK1URVbVzwjDcFtUjc4eq

npx hardhat lz:oapp:wire --oapp-config layerzero.config.ts --solana-secret-key <solana_secret_key> --solana-program-id C57J7hTDbQPLQWL7hwhHcDwnK1URVbVzwjDcFtUjc4eq


40287 linea sepolia
40168 solana devnet
40161 eth sepolia


npx hardhat --network sepolia send --dst-eid 40287 --amount 1000000000000000000 --to 0xc031C368b51c28266396273b0C6ce2489b00969d

https://layerzeroscan.com/tx/0x8e35197220b5b94c51bc7df7db0808affc7b3743f0615be340a21efd5396b5bd

npx hardhat --network sepolia send --dst-eid 40168 --amount 1000000000000000000 --to BhTpGbF7gBUiGjESQHVoBNhx2iy7Y21L2hAhRPdNT5ZF

https://testnet.layerzeroscan.com/tx/0xe219874be79cfe6153d4a982a332d8a906e7601ce709564c0cbd93da195783dd

npx hardhat lz:oft:solana:send --amount 1000000000 --from-eid 40168 --to 0xc031C368b51c28266396273b0C6ce2489b00969d --to-eid 40161 --mint 5my4h8qULDHJwxAVcKwk9n64gFz5dm3ridpzFTns6t8H --program-id C57J7hTDbQPLQWL7hwhHcDwnK1URVbVzwjDcFtUjc4eq --escrow 8MhP39SfLNWbLJVBCNNUU49z2CvRHfhqcj5uR57ZSWCZ

https://testnet.layerzeroscan.com/tx/z4LzEsaFMNQLTnTBnubghYGukcYqkCvL8bodageDDUvpScNRcVi1GRpkGXP9SL7PqCvwqwKcQDayDzmHqJnKnoF

npx hardhat --network linea_sepolia send --dst-eid 40161 --amount 1000000000000000000 --to 0xc031C368b51c28266396273b0C6ce2489b00969d

https://layerzeroscan.com/tx/0xf9938c70c5f34389c7fe104b28924e59a185d32c07a430a5a9ab617ceb3ccce6

```

@fb-alexcq @0xmer1in

### CODE REVIEW

File: layerzero.config.prod.ts [hunk 1]
```diff
@@ -76,23 +82,29 @@ const config: OAppOmniGraphHardhat = {
...
                 ],
                 sendConfig: {
                     executorConfig: {
-                        maxMessageSize: 99,
-                        executor: '0x0000000000000000000000000000000000000000',
+                        maxMessageSize: 10000,
+                        executor: '0x0408804C5dcD9796F22558464E6fE5bDdF16A7c7',
                     },
                     ulnConfig: {
                         confirmations: BigInt(1200),
```
Comment (0xMerl) [inline layerzero.config.prod.ts]: should be lower for linea -> ethereum

Comment (0xMerl) [reply layerzero.config.prod.ts]: same handle to ethereum -> linea receiveConfig

File: hardhat.config.prod.ts [hunk 2]
```diff
@@ -63,9 +63,9 @@ const config: HardhatUserConfig = {
             url: `https://rpc.ankr.com/eth`,
             accounts,
         },
-        merlin: {
-            eid: EndpointId.MERLIN_V2_MAINNET,
-            url: `https://rpc.merlinchain.io`,
```
Comment (0xMerl) [inline hardhat.config.prod.ts]: no need to delete?

**Comment by zhangchitc** [comment]
Thanks @0xmer1in both comments addressed

**Pull request merged**